webrtc.html

```diff
@@ -5597,8 +5597,9 @@ <h2>Methods</h2>
                 <code>InvalidStateError</code>.</li>
                 <li>If <code><var>parameters</var>.encodings.length</code>
                 is different from <var>N</var>, or if
-                any parameter in <var>parameters</var>, is
-                marked as a <dfn>Read-only parameter</dfn>, has a value that is
+                <code><var>parameters</var>.encodings</code> has been re-ordered,
+                or if any parameter in <var>parameters</var> is
+                marked as a <dfn>Read-only parameter</dfn> or has a value that is
```

Comment:
I think "or has a value" is meant to be "and has a value"? In other words, "is read-only *and* was changed"?

Reply:
Yes, agree.

Reply:
Might be better to rephrase - the sequence "A or B and C" isn't necessarily parsed as "A or (B and C)".